binr/rafind2/rafind2.c

```diff
@@ -39,7 +39,7 @@ static int hit(RSearchKeyword *kw, void *user, ut64 addr) {
 		return 0;
 	}
 	if (rad) {
-		printf ("f hit%d_%d 0x%08"PFMT64x" ; %s\n", 0, kw->count, addr, curfile);
+		printf ("f hit%d_%d 0x%08"PFMT64x" ; %s\n", 0, kw->count - 1, addr, curfile);
```

Comment:
this -1 scares me, may result in negative, producing an invalid flag name

Reply:
`count` is increased before the callback (set in `	r_search_set_callback (rs, &hit, buf);
) is called

Reply:
Make `count` increased after the callback. So no more `kw->count - 1`